.env.example

```diff
@@ -22,3 +22,4 @@ IPFS_PIN_MAX_PROMISES_PER_MINUTE=60
 POSTGRAPHILE_SERVE_PORT=6001
 NUMBER_OF_ERROR_RETRIES=5
 SOLANA_PROVIDER_ENDPOINT=https://solana-mainnet.g.alchemy.com/v2/key
+IPFS_NODE_MULTIADDRESS=/ip4/123.456.78.90/tcp/4001/ipfs/QmAbCdEfGhIjKlMnOpQrStUvWxYzAbCdEfGhIjKlMnOpQr,/ip4/123.456.78.90/tcp/4001/ipfs/QmAbCdEfGhIjKlMnOpQrStUvWxYzAbCdEfGhIjKlMnOpQr
```

Comment:
we expect a locally running ipfs node now, right? can you add this to README/docs